cloud-sql-proxy.yaml

```diff
@@ -23,6 +23,11 @@ pipeline:
       tag: v${{package.version}}
       expected-commit: 7cc6c8ebbc96de76e9962ba2a77aaf6534122774
 
+  - uses: go/bump
+    with:
+      deps: golang.org/x/crypto@v0.17.0
+      go-version: "1.21"
```

Comment:
is it just me or having to specify the go-version here is not ideal ? One more think to update in the future, how do we know the version to go here ?

Reply:
Just a general comment, it is not related to this PR.

Reply:
I agree - it's not ideal - I wonder if we could default to the version in the `go.mod` file instead?

Reply:
` I wonder if we could default to the version in the go.mod file instead?`

Indeed 🤩, that is how it should work. I wasn't sure why we have to set this property value 🤔.

Reply:
the go/bump defaults to 1.20 but when we use the apk go package I think that is the latest in this case 1.21, so we need to make explicit here that is using go1.21

i can be totally wrong as well